### DISCUSSION TITLE
Specifics label for every controls - layouts - converters

### DISCUSSION THREAD

**Discussion by Strypper** [CommunityToolkit/Maui#1832] [New Feature Discussions]
Can each controls - layers and coverters have it's own label in this repository?
![image](https://github.com/CommunityToolkit/Maui/assets/32596308/1e17e7a5-50c6-4ccb-883f-e1a7906ad738)
This picture is from the dotnet/maui repo

**Comment by bijington** [comment]
As maintainers we use the labels to know what area the issue affects and then the majority of issues with have the specific thing it affects in the title.

In intrigued, as a consumer of the toolkit why do you feel this change would be of value?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers. 

If you feel this is being closed in error please let us know and we can re-open.
